src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/ChunkContextUi.java

```diff
@@ -55,6 +55,41 @@ public ChunkContextUi(TextEditingTarget target, int renderPass,
       createToolbar(preambleRow);
    }
    
+   // Public static methods ---------------------------------------------------
+
+   public static String extractChunkLabel(String extractedChunkHeader)
```

Comment:
Realize you're just moving this, but adding a unit test for this would be helpful!

Reply:
Added.